website/docs/getting-started/04--contribution.md

```diff
@@ -1,8 +1,22 @@
 ---
 title: Contribution
-description: This is the (missing) long description of the component, that will come from the frontmatter attributes
+description: Contributing to the HashiCorp Design System
 ---
 
-## Lorem ipsum dolor
+## Overview
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Diam diam mi etiam mollis tortor vestibulum. Felis, arcu amet, nibh eget eget etiam orci aliquet. Varius facilisis magna faucibus commodo iaculis faucibus aliquet.
\ No newline at end of file
+Contributions to a design system take on many forms that can include:
+
+* Reporting bugs and giving feedback
+* Proposing new patterns
+* Improving our documentation
+* Participating in discussions or coming to Office Hours
+
+At this time we do not have a process defined for direct contributions of new components or patterns to the design system. However, we continue to listen to feedback as we consider the best way to tackle this problem.
```

Comment:
This is just for consideration, but putting what we don't accept first gets to the meat of the "problem" faster so consumers don't have to search for it.

```suggestion
While we don't accept direct contributions of new components or patterns, we do welcome contributions in the following forms:

* [Reporting bugs](https://docs.google.com/forms/d/e/1FAIpQLScpMXgrUTVT5fYriu4Pp48r4Nl_eCPluVnJLg0Yg3NXsRWvIA/viewform) and [giving feedback](https://docs.google.com/forms/d/e/1FAIpQLSdPShbSiUYgimppZeqWT99bVi6zLbKDThQUQTirH3RCj2okdg/viewform)
* [Proposing new components or patterns](https://docs.google.com/forms/d/e/1FAIpQLScpMXgrUTVT5fYriu4Pp48r4Nl_eCPluVnJLg0Yg3NXsRWvIA/viewform)
* Improving our documentation
* Participating in discussions or coming to Office Hours

We continue to listen to feedback as we consider the best way to open contributions further.
```